festim/geometry/mesh_1d.py

```diff
@@ -0,0 +1,72 @@
+from dolfinx import fem, mesh
+from mpi4py import MPI
+import ufl
+import numpy as np
+from festim import Mesh
+
+
+class Mesh1D(Mesh):
+    """
+    1D Mesh
+    Attributes:
+        vertices (list): the mesh x-coordinates
+        size (float): the size of the 1D mesh
+        V (dolfinx.fem.FunctionSpace): the function space of the simulation
```

Comment:
One mesh can be used in several simulations, therefore can have several function spaces, meaning the function space is not inherent to the mesh: it is not an attribute of the mesh (although you need a mesh to make a FunctionSpace)